install_sct.bat

```diff
@@ -12,31 +15,53 @@ FOR /F "tokens=*" %%G IN ('cd') do (SET PWD=%%G)
 cd %HOMEPATH%
 
 :: Download SCT and check out the branch requested by the user
+echo:
+echo ### Downloading SCT source code to %HOMEPATH%...
 git clone -b %git_ref% --single-branch --depth 1 https://github.com/spinalcordtoolbox/spinalcordtoolbox.git || goto :error
 
 :: Create and activate virtual environment to install SCT into
+echo:
+echo ### Using Python to create virtual environment...
 cd spinalcordtoolbox
 python -m venv venv_sct || goto :error
 CALL venv_sct\Scripts\activate.bat || goto :error
-
-:: Upgrade pip because default p3.7 pip won't resolve dependency conflicts correctly
-python -m pip install --upgrade pip || goto :error
+echo Virtual environment created and activated successfully!
 
 :: Install SCT and its requirements
+echo:
+echo ### Installing SCT and its dependencies...
+:: Pip needs to be upgraded because default p3.7 pip won't resolve dependency conflicts correctly
+python -m pip install --upgrade pip || goto :error
 pip install -r requirements.txt || goto :error
 pip install -e . || goto :error
 
 :: Install external dependencies
+echo:
+echo ### Downloading model files and binaries...
 FOR %%D IN (PAM50 gm_model optic_models pmj_models deepseg_sc_models deepseg_gm_models deepseg_lesion_models c2c3_disc_models binaries_win) DO sct_download_data -d %%D -k || goto :error
 
 :: Deactivate virtual environment
 deactivate
 
+:: Give further instructions that the user add the Scripts directory to their PATH
+echo:
+echo ### Installation finished!
+echo:
+echo To use SCT's command-line scripts in Command Prompt, please follow these instructions:
+echo:
+echo 1. Open the ^Start Menu -^> Type 'path' -^> Open 'Edit the System Environment Variables'

```

Comment:
TIL that Windows uses `^` as its escape char, as `\` is taken up for path separation. 🤔

Reply:
I think it's because [`start`](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/start) is a command in the batch language?

It's also possible that this is not actually necessary, aha. My IDE added syntax highlighting to `Start` and I felt the urge to escape it, but I can't remember if I double checked it or not.

Reply:
Yep, good call. `^Start` isn't necessary; `Start` is fine. :)